run-tests-jenkins

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install coverage
```

Comment:
I had this in only temporarily in the original script. It should be installed in development environments in the first place, rather than re-installed on every test run.